client/js/lounge.js

```diff
@@ -430,6 +430,14 @@ $(function() {
 		}
 	});
 
+	chat.on("click", ".condensed", function() {
+		$(this).toggleClass("closed");
+	});
+
+	chat.on("click", ".condensed div", function(e) {
```

Comment:
I don't have time for this before I go. You can either fix it yourself or we can just leave it. Sorry.